interface-definitions/service-router-advert.xml.in

```diff
@@ -249,6 +249,18 @@
                       <valueless/>
                     </properties>
                   </leafNode>
+                  <leafNode name="deprecate-prefix">
+                    <properties>
+                      <help>Deprecate the prefix on shutdown</help>
```

Comment:
Please update the help to: `Upon shutdown, this option will deprecate the prefix by announcing it in the shutdown RA`